.github/workflows/populate-metabase-itou.yml

```diff
@@ -47,9 +47,10 @@ jobs:
       run:
         echo "CRON_TASK_APP_NAME=`echo \"c1-cron-daily-$(date +%y-%m-%d-%Hh-%M)"`" >> $GITHUB_ENV
 
-    - name: 🏷 Setup worker command
-      run:
+    - name: 🏷 Setup worker to run django command
+      run: |
         echo 'CC_WORKER_COMMAND=./manage.py populate_metabase_itou --verbosity 2' >> $GITHUB_ENV
+        echo 'CC_WORKER_RESTART=no' >> $GITHUB_ENV
```

Comment:
```suggestion
        # By default, workers always restart on failure. We want to avoid an infinite loop if it happens.
        echo 'CC_WORKER_RESTART=no' >> $GITHUB_ENV
```